1.8/install/database-restore.md

```diff
@@ -0,0 +1,143 @@
+
+In the following instructions Linux CentOS 7 will be used, so commands may differ.
+
+## MySQL Database Restore
+
+1. Once logged into PhpMyAdmin, select your database from the side-menu on the left. 
+
+[![database screen](https://i.gyazo.com/f4376fa46113a79a691d023ee07bf430.png)](https://i.gyazo.com/f4376fa46113a79a691d023ee07bf430.png)
+
+2. From here, select "Import" from the top menu. You will be presented with a page like this:
+
+[![import screen](https://i.gyazo.com/d460da5ba195ecc8f6b0056a8d01b0f0.png)](https://i.gyazo.com/d460da5ba195ecc8f6b0056a8d01b0f0.png)
+
+3. Select "Choose File", browse to your backup, and click "Open" on the Windows Prompt. 
+
+4. Generally, the settings on this page can be left as is, but you may alter where appropriate.
+
+5. Click "Go". If successful, a message saying so will be presented as well as your tables in the side-menu on the left.
+
+## SQLite Database Restore
+
+1. Login to PhpLiteAdmin and select your database from the side-menu on the left. 
+
+[![phpliteadmin screen](https://i.gyazo.com/977c411c55aceb763dae6b524323caf4.png)](https://i.gyazo.com/977c411c55aceb763dae6b524323caf4.png)
+
+2. Select "Import" from the top menu and you will be presented with a page like this:
+
+[![phpliteadmin import](https://i.gyazo.com/d8bf3f0c277aaf6d38295adfcd364b8b.png)](https://i.gyazo.com/d8bf3f0c277aaf6d38295adfcd364b8b.png)
+
+3. Select "Choose File", browse to your backup, and click "Open" on the Windows Prompt.
+
+4. Generally, the settings on this page can be left as is, but you may alter where appropriate.
+
+5. Click "Import". If successful, you will see your tables in the side-menu on the left, as well as a message letting you know the import was successful.
+
+[![phpliteadmin success](https://i.gyazo.com/0aafbe3053692a90f114125deb21bded.png)](https://i.gyazo.com/0aafbe3053692a90f114125deb21bded.png)
+
+## PostgreSQL Database Restore
+
+1. Login to PhpPgAdmin and select your database from the side-menu on the left.
+
+[![phppgadmin screen](https://i.gyazo.com/26cc40c5ed671cd2f0ff075da275df6b.png)](https://i.gyazo.com/26cc40c5ed671cd2f0ff075da275df6b.png)
+
+2. Select "SQL" from the top menu and you will be presented with a page like this:
+
+[![phppgadmin import](https://i.gyazo.com/9bba8dde27d530ec52bb001ece08018e.png)](https://i.gyazo.com/9bba8dde27d530ec52bb001ece08018e.png)
+
+3. Select "Choose File", browse to your backup, and click "Open" on the Windows Prompt.
+
+4. Click "Execute". Allow the SQL to run. If successful, you will be able to scroll down to the bottom of the page and it will look something like this:
+
+[![phppgadmin successful](https://i.gyazo.com/5c74e92bb59bf5d2cf0c4afa4ac5e50d.png)](https://i.gyazo.com/5c74e92bb59bf5d2cf0c4afa4ac5e50d.png)
+
+## Database Restore using SSH
+
+### MySQL
+
+1. After logging into your server via SSH, run the following:
+
+``` mysql -u YOUR_MYBB_DB_USER -p YOUR_MYBB_DBNAME < /PATH/TO/DUMP.SQL ``` 
```

Comment:
This should be written as:

    ```
    mysql -u YOUR_MYBB_DB_USER -p YOUR_MYBB_DBNAME < /PATH/TO/DUMP.SQL
    ```

Note the difference in formatting when viewed.